data/json/recipes/basecamps/recipe_modular_field_log.json

```diff
@@ -0,0 +1,337 @@
+[
+  {
+    "type": "recipe",
+    "result": "faction_base_modular_hub_room0_log_northeast",
+    "description": "We need some shelter, so build half of a log shack with a wooden roof on the northeast side of the camp",
+    "category": "CC_BUILDING",
+    "subcategory": "CSC_BUILDING_BASES",
+    "autolearn": false,
+    "never_learn": true,
+    "construction_blueprint": "fbmh_room0_log_northeast",
+    "blueprint_name": "northeast shack",
+    "blueprint_requires": [ { "id": "fbmh_0" } ],
+    "blueprint_provides": [ { "id": "fbmh_northeast" } ],
+    "blueprint_excludes": [ { "id": "fbmh_northeast" } ],
+    "blueprint_autocalc": true
```

Comment:
I think `blueprint_autocalc` is not required anymore.

Reply:
I leave it out or add in something else?

Reply:
Just omit it.  Any blueprint that lacks `blueprint_needs` and doesn't have `check_blueprint_needs: false` will have its needs auto-calculated.

Reply:
Will do, thanks.  I didn't see that mentioned in your PR.

Reply:
Yeah, sorry.  Didn't mention it in the PR.  Was going to put it in the docs but then I didn't write them.